cafy_pytest/plugin.py

```diff
@@ -328,6 +338,7 @@ def _requests_retry(logger, url, method, data=None, files=None,  headers=None, t
 
 @pytest.hookimpl(tryfirst=True)
 def pytest_configure(config):
+    print (f"config: {config}")
```

Comment:
Hopefully this will be reverted before merging? This would cause extra output spam for cafy users.

(Might be safer to keep the PR as a draft if it's not ready to be merged).

Reply:
Yes, that was for internal testing. I have prepended the PR title with "DO NOT MERGE"